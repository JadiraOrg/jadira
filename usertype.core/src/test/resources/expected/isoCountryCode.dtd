<!ELEMENT dataset (
    ISOCOUNTRYCODE*)>

<!ELEMENT ISOCOUNTRYCODE EMPTY>
<!ATTLIST ISOCOUNTRYCODE
    ID CDATA #REQUIRED
    ISOCOUNTRYCODE CDATA #IMPLIED
    NAME CDATA #IMPLIED
>

