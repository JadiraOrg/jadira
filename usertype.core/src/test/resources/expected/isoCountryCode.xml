<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE dataset SYSTEM "isoCountryCode.dtd">
<dataset>
  <ISOCOUNTRYCODE ID="0" ISOCOUNTRYCODE="AD" NAME="test_0"/>
  <ISOCOUNTRYCODE ID="1" ISOCOUNTRYCODE="GB" NAME="test_1"/>
  <ISOCOUNTRYCODE ID="2" ISOCOUNTRYCODE="US" NAME="test_2"/>
  <ISOCOUNTRYCODE ID="3" ISOCOUNTRYCODE="FR" NAME="test_3"/>
  <ISOCOUNTRYCODE ID="4" NAME="test_4"/>
</dataset>